### PR TITLE
KAFKA-12246: Remove redundant suppression in KafkaAdminClient

### PR DESCRIPTION
**Pull request by wycccccc**
issue: https://issues.apache.org/jira/browse/KAFKA-12246
Remove redundant comments in KafkaAdminClient.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Review by chia7712 — APPROVED** [review]
@wycccccc Thanks for your patch. LGTM

nit: you can file a minor PR without existent jira if the PR is trivial. see https://cwiki.apache.org/confluence/display/KAFKA/Contributing+Code+Changes for more details.

**Comment by wycccccc** [comment]
@chia7712  Thanks for your review. I will go hard.

File: clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java [hunk 1]
```diff
@@ -2411,8 +2411,7 @@ public DescribeLogDirsResult describeLogDirs(Collection<Integer> brokers, Descri
                     // Query selected partitions in all log directories
                     return new DescribeLogDirsRequest.Builder(new DescribeLogDirsRequestData().setTopics(null));
                 }
-
-                @SuppressWarnings("deprecation")
+
```
Comment (chia7712) [inline clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java]: please remove this change (unnecessary Indent)

Comment (wycccccc) [reply clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java]: Thax remind,I have remove.

**Pull request merged**